buildifier/buildifier.go

```diff
@@ -77,6 +77,14 @@ if the -v flag is given, prints their names to standard error.
 In print_if_changed mode, buildifier shows the file contents it would write.
 The default mode is fix. -d is an alias for -mode=diff.
 
+The lint flag selects the lint mode to be used: off, warn, fix.
```

Comment:
Looks like line 60 also needs to be updated, it doesn't mention lint at all.

Reply:
https://github.com/bazelbuild/buildtools/pull/467